pytorch_lightning/loggers/base.py

```diff
@@ -172,18 +158,23 @@ def __getitem__(self, index: int) -> LightningLoggerBase:
     def experiment(self) -> List[Any]:
         return [logger.experiment for logger in self._logger_iterable]
 
+    @rank_zero_only
```

Comment:
@Borda i don't know about these changes....
@neggert dealt with this  a long time ago and likely left  these as non-rank zero for a reason...
I don't even think loggers are instantiated in non-zero processes